### PR TITLE
Messaging: maintainers

### PR DESCRIPTION
**Pull request by bkeepers**
This pulls out just the messaging for maintainers from #13 so we can work through it.

Ignore the specific phrasing of this doc for now, but let's focus on the high-level ideas:

- [x] Who is the audience?
- [x] Why should they care?
- [x] What specific steps should they take to get started?

cc @MikeMcQuaid @nayafia @kytrinyx #16 #17

### CODE REVIEW

File: app/views/home/maintainers.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
```
Comment (nayafia) [inline app/views/home/maintainers.md]: * that are looking for contributions

(aka why are they here on this page right now?)

File: app/views/home/maintainers.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
```
Comment (nayafia) [inline app/views/home/maintainers.md]: We should get more explicit about their pain point. They don't necessarily care if it's "welcoming", they just want contributions.

_"Take one day this week to make your project easy to contribute to"_

File: app/views/home/maintainers.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+> Why should I care?
+
+You could use help
```
Comment (nayafia) [inline app/views/home/maintainers.md]: You want contributions. People aren't contributing (or not in the ways you want). Give your project some love and TLC to make it easy for others to contribute.

File: app/views/home/maintainers.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+> What are the reasons I should care about getting more contributors for my open source project?
```
Comment (nayafia) [inline app/views/home/maintainers.md]: I think the question here isn't "why should I care about getting more contributors" (we should assume they're on this page bc they want them), but "why should I spend all this extra time on a project that is already a PITA"

AKA how does cleaning up my project attract more contributors?

Comment (nayafia) [inline app/views/home/maintainers.md]: hmm. I'm reconsidering. Maybe it's "How does getting more contributors reduce my workload?"

File: app/views/home/maintainers.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+- **Reduce your workload** - Regular contributors help
```
Comment (nayafia) [inline app/views/home/maintainers.md]: These bullets should be revised in light of the above, but I'm not exactly sure what to put here.

Somehow I'm picturing easy CTAs more than benefits, like:

* Document how people can help: well-documented projects get contributors blahblah
* Close old issues: a fresh issue tracker makes you and them feel good  blah
* Label first timer issues: make it easy for newcomers to scan blah

File: app/views/home/maintainers.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+### Set aside time
+
+TODO: words about what might block you (time, boss, etc)
+
+CTA: Add to calendar
+
+> Knowledge: What if I feel like I don't know enough yet?
+
+## Learn how to build a welcoming community
```
Comment (nayafia) [inline app/views/home/maintainers.md]: Might be better to link to maintainer best practices (feels like less pressure than "build a community")

File: app/views/home/maintainers.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Building a community that encourages people to use, contribute to, and evangelize your project.
+
+CTA: [Read the guide](https://opensource.guide/building-community/)
+
+> Onramp: I'm scared, can you hold my hand?
+
+## ????
```
Comment (nayafia) [inline app/views/home/maintainers.md]: Suggest moving this section into the bullets above

File: app/views/home/maintainers.md (hunk 2 above)
Comment (kytrinyx) [inline app/views/home/maintainers.md]: My first thought when I read _make your project easy to contribute to_ was "that's impossible".

Maybe s/easy/easier/

_"Take one day this week to make your project easier to contribute to"_

File: app/views/home/maintainers.md (hunk 3 above)
Comment (kytrinyx) [inline app/views/home/maintainers.md]: > People aren't contributing (or not in the ways you want)

I think this is the key thing. I was headed in the direction of "spread the burden / avoid burnout" but both of those are very negative, whereas getting more and more useful contributions is positive.

File: app/views/home/maintainers.md (hunk 4 above)
Comment (kytrinyx) [inline app/views/home/maintainers.md]: My biggest questions as a maintainer (especially about a year ago) was around "yeah, I get that I should delegate, I get that I should make it easier for people to contribute, and dammit I'm trying, but I just can't figure out _what_ specific things I can do to make it so". So if this promises to point me at specific things that I could do that would help, I would be sold.

File: app/views/home/maintainers.md (hunk 2 above)
Comment (nayafia) [inline app/views/home/maintainers.md]: I'm tempted for the CTA here to just be "give your project some love" or some such but it feels too vague.

File: app/views/home/maintainers.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+- ensure your project's README file details whether your project wants contributions, the project's scope and how to contribute to it.
+- improve onboarding docs (missing CONTRIBUTING, CODE_OF_CONDUCT, etc.)
+- diagnose community problems? (e.g. fossheartbeat)
+- create first timer issues
```
Comment (kytrinyx) [inline app/views/home/maintainers.md]: Make documentation for specific contribution cases to make it possible for people to come in and look for how to do exactly what they're looking for, and ignore irrelevant information (e.g. on Exercism that would be: fix a typo in an exercise README, or add a new exercise, or port an existing exercise to a new programming language, launch a new language track, extract canonical test inputs/outputs for an exercise).

File: app/views/home/maintainers.md (hunk 1 above)
Comment (bkeepers) [inline app/views/home/maintainers.md]: I think this is pretty valuable to people that haven't realized that they should be optimizing their project for new contributors.

File: app/views/home/maintainers.md (hunk 5 above)
Comment (MikeMcQuaid) [inline app/views/home/maintainers.md]: Agreed. Also: automating manual workflows, documenting and reducing scope and optimising for the 99% not the 100%.

**Comment by MikeMcQuaid** [comment]
Closing in favour of copy in https://github.com/ossfriday/ossfriday/pull/28.

**Pull request closed without merging**